docs/docs/developerguide/README.md

```diff
@@ -304,3 +318,44 @@ This package includes the script linked in the documentation above, which simpli
   DJANGO_GMAIL_API_CLIENT_SECRET='google_assigned_secret'
   DJANGO_GMAIL_API_REFRESH_TOKEN='google_assigned_token'
   ```
+
+
+#### Teamware Privacy Policy and Terms & Conditions
+
+Teamware includes a default privacy policy and terms & conditions, which are required for running the application.
+
+The default privacy policy is compliant with UK GDPR regulations, which may comply with the rights of users of your deployment, however it is your responsibility to ensure that this is the case.
```

Comment:
I'd say "is intended to be compliant" or "is believed to be compliant" rather than "**is** compliant", unless we've actually had it checked by someone with legal training...  Otherwise it looks like we're providing legal advice, which we're not competent to do.